docs/devel/how-to-doc.md

```diff
@@ -0,0 +1,171 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/devel/how-to-doc.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+Document Conventions
+====================
+
+Updated: 11/3/2015
+
+*This document is oriented at users and developers who want to write documents for Kubernetes.*
+
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+
+  - [What Are Mungers?](#what-are-mungers)
+  - [Table of Contents](#table-of-contents)
+  - [Writing Examples](#writing-examples)
+  - [Adding Links](#adding-links)
+  - [Auto-added Mungers](#auto-added-mungers)
+    - [Unversioned Warning](#unversioned-warning)
+    - [Is Versioned](#is-versioned)
+    - [Generate Analytics](#generate-analytics)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
+## What Are Mungers?
+
+Mungers are like gofmt for md docs which we use to format documents. To use it, simply place
+
+```
+<!-- BEGIN MUNGE: xxxx -->
```

Comment:
One only needs to know this when writing a new munger. I'd put this at the end of the doc. Most people need to know:

1. That we munge the docs and how to do it (update-generated-docs.sh)
2. That we verify the docs have been munged
3. How to write links
4. How to get a TOC
5. How to include an example
6. Other misc. doc conventions, such as when to use triple vs. single backticks, console vs. shell quote format, whether to use one `#` for the doc title or equal signs, etc.